public/js/kinesis-example.js

```
@@ -14,38 +14,34 @@ AWS.config.credentials.get(function(err) {
+            const scrollHeight = scrollableElement.scrollHeight;  // body full height.
+            const scrollTop = scrollableElement.scrollTop;  // current scroll position from top.
 
-            var scrollTopPercentage = Math.round((scrollTop / scrollHeight) * 100);
-            var scrollBottomPercentage = Math.round(((scrollTop + scrollableHeight) / scrollHeight) * 100);
+            const scrollTopPercentage = Math.round((scrollTop / scrollHeight) * 100);
+            const scrollBottomPercentage = Math.round(((scrollTop + scrollableHeight) / scrollHeight) * 100);
 
             // Create the Amazon Kinesis record
-            var date = new Date();
+            const date = new Date();
             date = date.toISOString().replace('T', ' ').slice(0, 19)  // convert to "yyyy-mm-dd HH:MM:SS" format. 
```

Biome

[error] Can't assign date because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)